<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>Port Adı:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="baudLabel">
          <property name="text">
           <string>Baud Rate:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="paityLabel">
          <property name="text">
           <string>Parity:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="dataLabel">
          <property name="text">
           <string>Data Bits:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="stopLabel">
          <property name="text">
           <string>Stop Bits:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="flowLabel">
          <property name="text">
           <string>Flow Kontrolü:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="portCombo"/>
        </item>
        <item>
         <widget class="QComboBox" name="baudCombo">
          <property name="currentIndex">
           <number>3</number>
          </property>
          <item>
           <property name="text">
            <string>1200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>57600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>-1</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="parityCombo">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>NoParity</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EvenParity</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OddParity</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SpaceParity</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MarkParity</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UnknownParity</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="dataCombo">
          <property name="currentIndex">
           <number>3</number>
          </property>
          <item>
           <property name="text">
            <string>Data5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Data6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Data7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Data8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UnknownDataBits</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="stopCombo">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>OneStop</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OneAndHalfStop</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TwoStop</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UnknownStopBits</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="flowCombo">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>NoFlowControl</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HardwareControl</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SoftwareControl</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UnknownFlowControl</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="statusLabel">
            <property name="text">
             <string>Durum</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="radioGroupBox">
        <property name="title">
         <string>Gönderim Tipi</string>
        </property>
        <widget class="QRadioButton" name="hexRadioButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>54</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Hex</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="stringRadioButton">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>20</y>
           <width>82</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>String</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <zorder>hexRadioButton</zorder>
        <zorder>stringRadioButton</zorder>
        <zorder>hexRadioButton</zorder>
        <zorder>stringRadioButton</zorder>
        <zorder>hexRadioButton</zorder>
        <zorder>stringRadioButton</zorder>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="messageLineEdit">
        <property name="toolTip">
         <string/>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Mesaj Giriniz...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Mesajı Gönder</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="openButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Aç</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeButton">
          <property name="text">
           <string>Kapat</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
